imports/api/questions.js

```diff
@@ -38,6 +38,8 @@ const questionPattern = {
   courseId: Match.Maybe(Helpers.MongoID),
   // student submitted questions are always public, prof can mark question templates as public
   public: Boolean,
+  solution: String,
```

Comment:
These should be match Maybe, for backwards compatability. That means, when checking to display a solution, you have to make sure the solution actually exists.

Also, add comments (e.g. solution is the html), make it clear what the difference between an answer is (A) versus a solution (the answer is A because...)